app/views/editor/poll/PollEditView.coffee

```diff
@@ -41,6 +41,10 @@ module.exports = class PollEditView extends RootView
 
   onLoaded: ->
     super()
+
+    if @poll.get('answers') == undefined
+      @poll.set('hidden', true)
```

Comment:
Curious why this condition is needed?

Reply:
This manually sets the hidden setting on new polls, to make sure they are initially created in a hidden state. This is to reduce errors of saving a poll and having it immediately appear in production.

This is because polls need to be saved and then linked together to make the multi-poll setup. It's a safety feature.

Reply:
So is it assuming that as soon as answers are configured, we show them to prod users? Or will we need to manually remove the hidden property using the editor?

Reply:
This property is manual. We would only change it in the editor on the poll we want automatically shown to the user once everything is set up.
Hidden stops the automatic poll system from displaying the poll to the user. But the poll can still be explicitly fetched.

I can make this more explicit in the guide with a "how to deploy poll to production section".